packages/angular/cli/commands/update-impl.ts

```diff
@@ -307,6 +307,19 @@ export class UpdateCommand extends Command<UpdateCommandSchema> {
       );
     }
 
+    if (options.all) {
+      const updateCmd = this.packageManager === PackageManager.Yarn
+        ? `'yarn upgrade-interactive' or 'yarn upgrade'`
+        : `'${this.packageManager} update'`;
+
+      this.logger.warn(`
+        '--all' functionality has been removed removed as updating multiple packages at once is not recommended.
+        To update the depencencies in your workspace 'package.json' use ${updateCmd} instead.
```

Comment:
```suggestion
        To update the non-Angular dependencies in your workspace 'package.json' use ${updateCmd} instead.
```

Should we mention to run the package manager update after using `ng update` for Angular dependencies?

Reply:
non-Angular dependencies might imply packages which are not under the `@angular` scope but still has migrations.

Maybe reword to `to update packages which don’t provide ng update capabilities`

Reply:
sounds good

Reply:
Yeah we should. I’ll think of a way to word that

Reply:
PTAL